content/en/platform/corda/5.0/developing-applications/packaging/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+date: '2023-02-23'
+title: "Packaging"
+project: corda
+version: 'Corda 5.0'
+menu:
+  corda5:
+    identifier: corda5-develop-packaging
+    parent: corda5-develop
+    weight: 4000
+section_menu: corda5
+---
+Just like a regular application, your CorDapp must be packaged for distribution and installation. Corda takes a three-layered model to its packaging design to allow maximum reusability and portability:
+
+1. **Corda Package (CPK):** Represents a single code-entity authored by a CorDapp developer. CPKs are the Corda equivalent of a software library. They represent testable, reusable, sub-components of a final application. Corda runs each CPK in its own sandbox, isolated from other CPKs. See [Build a CPK]({{< relref "./cpk.md">}}).
```

Comment:
Not sure about:
> Corda runs each CPK in its own sandbox, isolated from other CPKs

To some extent it is true, but then one CPK can depend on another so they would not be isolated in that case.
I don't want to ruin the simplicity with detail.
Is it ok that this is not 100% true once you get into detail?

Reply:
No, from what you're saying, it's misleading so I should change it. I suggest we:

- Just remove the sentence "Corda runs each CPK in its own sandbox, isolated from other CPKs."
OR
- Change "Corda runs each CPK in its own sandbox, isolated from other CPKs." to "Where no dependencies occur, Corda can run each CPK in its own sandbox, isolated from other CPKs." (or similar)

Reply:
I'm happy with either suggestion. I think removing the sentence fits the brief nature of this part while adding more detail changes the feel.